components/style/properties/helpers.mako.rs

```diff
@@ -452,31 +452,26 @@
             }
             pub fn parse_declared<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
                                           -> Result<PropertyDeclaration, ParseError<'i>> {
-                match input.try(|i| CSSWideKeyword::parse(context, i)) {
-                    Ok(keyword) => Ok(PropertyDeclaration::CSSWideKeyword(LonghandId::${property.camel_case}, keyword)),
-                    Err(_) => {
-                        input.look_for_var_functions();
-                        let start = input.position();
-                        let specified = parse_specified(context, input);
-                        if specified.is_err() {
-                            while let Ok(_) = input.next() {}  // Look for var() after the error.
-                        }
-                        let var = input.seen_var_functions();
-                        if specified.is_err() && var {
-                            input.reset(start);
-                            let (first_token_type, css) =
-                                ::custom_properties::parse_non_custom_with_var(input)?;
-                            return Ok(PropertyDeclaration::WithVariables(LonghandId::${property.camel_case},
-                                                                         Arc::new(UnparsedValue {
-                                css: css.into_owned(),
-                                first_token_type: first_token_type,
-                                url_data: context.url_data.clone(),
-                                from_shorthand: None,
-                            })))
-                        }
-                        specified.map(|s| PropertyDeclaration::${property.camel_case}(s))
-                    }
+                input.look_for_var_functions();
+                let start = input.position();
+                let specified = parse_specified(context, input);
+                if specified.is_err() {
+                    while let Ok(_) = input.next() {}  // Look for var() after the error.
+                }
+                let var = input.seen_var_functions();
+                if specified.is_err() && var {
+                    input.reset(start);
+                    let (first_token_type, css) =
+                        ::custom_properties::parse_non_custom_with_var(input)?;
+                    return Ok(PropertyDeclaration::WithVariables(LonghandId::${property.camel_case},
+                                                                 Arc::new(UnparsedValue {
+                        css: css.into_owned(),
+                        first_token_type: first_token_type,
+                        url_data: context.url_data.clone(),
+                        from_shorthand: None,
+                    })))
                 }
+                specified.map(|s| PropertyDeclaration::${property.camel_case}(s))
```

Comment:
nit: I think you can just remove the closure (`specified.map(PropertyDeclaration::${property.camel_case})`)